diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/DiktatRuleSetProvider.kt

```diff
@@ -20,7 +20,7 @@ import org.slf4j.LoggerFactory
  */
 const val DIKTAT_RULE_SET_ID = "diktat-ruleset"
 
-class DiktatRuleSetProvider(private val jsonRulesConfig: String = "rules-config.json") : RuleSetProvider {
+class DiktatRuleSetProvider(private val jsonRulesConfig: String = "../diktat-analysis.yml") : RuleSetProvider {
```

Comment:
No, don't change location, it worked more or less fine as it is.

Reply:
What was your intention? Currently we are reading config file either from working directory or from resources root.